cdci_data_analysis/analysis/parameters.py

```diff
@@ -382,30 +389,26 @@ def value(self):
 
     @value.setter
     def value(self, v):
-        if v is not None and v!='':
+        if v is not None and v != '':
             self.check_float_value(v, name=self.name, units=self.units)
             self._v = np.float(v)
 
         else:
             self._v=None
 
+    def get_value_in_default_format(self):
+        return float(self.value)
+
     @staticmethod
     def check_float_value(value, units=None, name=None):
         #print('check type of ',name,'value', value, 'type',type(value))
-        if value is None or value=='':
+        if value is None or value == '':
             pass
         else:
             try:
-              value=ast.literal_eval(value)
+                value = float(value)
```

Comment:
it does the same as `float(value)`, without assignment. Let's either use new `value` or not assign it.